lib/Gemfile

```diff
@@ -17,33 +17,34 @@ gem "activesupport", :git => "git://github.com/rails/rails.git", :branch => "4-2
 gem "activerecord",  :git => "git://github.com/rails/rails.git", :branch => "4-2-stable"
 
 # Not locally modified and not required
-gem "awesome_spawn",        "~> 1.3",        :require => false
-gem "binary_struct",        "~> 2.0",        :require => false
-gem "excon",                "~>0.40",        :require => false
-gem "ezcrypto",             "=0.7",          :require => false
-gem "facade",               "~>1.0.5",       :require => false  # Used by util/pathname2.rb
-gem "ffi",                  "~>1.9.3",       :require => false
-gem "ffi-vix_disk_lib",     "~>1.0.1",       :require => false  # used by lib/VixDiskLib
-gem "fog",                  "~>1.29.0",      :require => false
-gem "httpclient",           "~>2.5.3",       :require => false
+gem "awesome_spawn",        "~> 1.3",         :require => false
+gem "binary_struct",        "~> 2.0",         :require => false
+gem "excon",                "~>0.40",         :require => false
+gem "ezcrypto",             "=0.7",           :require => false
+gem "facade",               "~>1.0.5",        :require => false  # Used by util/pathname2.rb
+gem "ffi",                  "~>1.9.3",        :require => false
+gem "ffi-vix_disk_lib",     "~>1.0.1",        :require => false  # used by lib/VixDiskLib
+gem "fog",                  "~>1.29.0",       :require => false
+gem "fog-core",             "!=1.31.1",       :require => false
+gem "httpclient",           "~>2.5.3",        :require => false
 gem "linux_admin",          ">=0.10.1", "<1", :require => false
-gem "log4r",                "=1.1.8",        :require => false
-gem "memoist",              "~>0.11.0",      :require => false
-gem "memory_buffer",        ">=0.1.0",       :require => false
-gem "more_core_extensions", "~>1.2.0",       :require => false
-gem "nokogiri",             "~>1.6.0",       :require => false
+gem "log4r",                "=1.1.8",         :require => false
+gem "memoist",              "~>0.11.0",       :require => false
+gem "memory_buffer",        ">=0.1.0",        :require => false
+gem "more_core_extensions", "~>1.2.0",        :require => false
```

Comment:
@brandondunne if you're going to line up whitespace, can you pad it more for each column since gem name and version has no room to grow?

I'd rather not have to deal with git blame/log pointing to a series of one-two space whitespace changes each time.